importer/importer_test.go

```diff
@@ -0,0 +1,245 @@
+package importer
+
+import (
+	"errors"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestAppendSearchFromEnv(t *testing.T) {
+	i := Importer{}
+	os.Setenv("IMPORTER_TEST", "//example.com/path:/local/path")
```

Comment:
`os.Setenv` (and `os.Unsetenv`) can fail, so `require.NoError`?

Reply:
Done in 38a81ffbe14da36daf32f9c5ec5621edda514a80